code/scripting/api/objs/sound.cpp

```diff
@@ -124,6 +124,15 @@ ADE_FUNC(isValid, l_SoundEntry, NULL, "Detects whether handle is valid", "boolea
 	return ade_set_args(L, "b", seh->IsValid());
 }
 
+ADE_FUNC(tryLoad, l_SoundEntry, nullptr, "Detects whether handle references a sound that can be loaded", "boolean", "true if a load succeeded, false if not, nil if a syntax/type error occurs")
```

Comment:
Just curious what the motivation was for adding this function?

Reply:
Just to match the `isValid` function above it.  In this use case, `isValid` does not do what a user might think it does.